.github/workflows/go-test.yaml

```diff
@@ -18,10 +18,10 @@ jobs:
           key: ${{ runner.os }}-go-test-${{ secrets.CACHE_VERSION }}-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-test-
-      - name: Create e2e run test config
+      - name: Create Test config
         shell: bash
         run: |
-          echo "${{ secrets.E2E_TEST_CONFIG }}" > ./e2e/config/application.yml
+          echo "${{ secrets.E2E_TEST_CONFIG }}" > ./config/application.yml
```

Comment:
Currently, this is done because fee tests depend on the configuration.
I would suggest a manual mock of the configuration - declaring a `var` config with the needed addresses and fee params.